internal/command/deploy/deploy.go

```diff
@@ -215,6 +215,14 @@ func New() *Command {
 			Description: "Do not run the release command during deployment.",
 			Default:     false,
 		},
+		flag.String{
+			Name:        "export-manifest",
+			Description: "Specify a file to export the deployment configuration to a deploy manifest file, or '-' to print to stdout.",
+		},
+		flag.String{
+			Name:        "manifest",
+			Description: "Path to a deploy manifest file to use for deployment.",
```

Comment:
Is it also worth mentioning in the Description the use of "-" for getting manifest from  to io.In, as can be seen in this [case "-"](https://github.com/superfly/flyctl/pull/3769/files#diff-f1ed021d538250bb75f4922951ea5c467ce53115b46483d10c198a5c88c9cc60R282)?

Also! I'm curious! I can't imagine how  the scenario for using  "-" actually works( I want to test it out). Is it like this?
1. run `fly deploy --export-manifest="-"`, will print manifest details to stdout 
2. run `fly deploy --manifest="-"` from ( somewhere else(a different console or machine?) OR the same console/machien? ) to get the manifest stream from running step #1 above

Reply:
Heads up I changed `--manifest` to `--from-manifest`. I think `fly deploy --from-manifest` is better equipped to convey that we are deploying using a manifest. 

And the right format is `fly deploy --from-manifest -` and `fly deploy --export-manifest`. This is mostly because flag.GetString always returns `""` even when the flag is not specified. The `if flag.GetString(ctx, "from-manifest")=="" would always be true and take over everything if we decided that an absence of the manifest filename means we will be reading from StdIn. So I decided to use this pattern cause I saw it used inf `kubectl`

Reply:
We can still improve the docs though